### ISSUE TITLE
Debugger Points To Incorrect Code Line in Base GDScript File

### ISSUE THREAD

**Issue by SolarLune** [godotengine/godot#1109]
Really verbose title, but it's not too difficult to understand. Basically, you have a "base" file and you require an instance of a class from an external file. If that class references a variable that doesn't exist, the debugger will point to the error in the line number of the class, but in the "base" GDScript file. The error will be: `invalid set index 'keycode' (on base: 'Reference ()").` 

Here are the example script files:

The base .gd file:
```
extends Spatial

var other = preload("res://external/input.gd")
	
func _ready():
	
	var input_key = other.InputKey.new(other.IT_KEY, "A")
```

And the "input.gd" file.
```
const IT_KEY = 0

class InputKey:

	var input_type
	#var keycode  # What's necessary to make it work
	
	func _init(input_type, keycode):

		self.input_type = input_type
		self.keycode = keycode
```

**Comment by akien-mga** [comment]
I can confirm the bug in the current master branch, dde6396.

**Comment by akien-mga** [comment]
@reduz As you're improving the debugger/adding a profiler, it would be nice to get a fix for this one.

**Comment by akien-mga** [comment]
Not critical for the upcoming 2.1, so moving to the next milestone.

**Comment by vnen** [comment]
I'm pretty sure this was already fixed.

**Comment by vnen** [comment]
Yeah, I cannot reproduce this. I assume it was fixed by 3754f6c.
